sw/airborne/subsystems/imu/imu_mpu9250_spi.c

```diff
@@ -194,6 +204,7 @@ void imu_mpu9250_event(void)
     imu_scale_mag(&imu);
     AbiSendMsgIMU_MAG_INT32(IMU_MPU9250_ID, now_ts, &imu.mag);
```

Comment:
Since you check for the overflow status bit above, the measurement should also not be sent via ABI if not valid...